activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+*   Add `allow_retry` to `sql.active_record` instrumentation.
+
+    This enables identifying queries which are and are not automatically
```

Comment:
@skipkayhil There is some missing word in this sentence `...which are ? and are not...`

Reply:
26d23be18a54b984046393a69a1aaa56baa1b302

Reply:
Thanks! The original intention was `which {are, are not} automatically retryable` but I agree that the phrasing was a bit confusing